loadgen/mlperf.conf

```diff
@@ -35,3 +35,5 @@ ssd.*.performance_sample_count_override = 128
 bevformer.ConstantStream.target_qps = 12
 deeplabv3plus.ConstantStream.target_qps = 15
 ssd.ConstantStream.target_qps = 15
+
+bevformer.*.group_sizes = 39, 40, 40, 39, 39, 39, 40, 40, 40, 40, 40, 39, 40, 40, 40, 40, 40, 40, 40, 40, 40, 40, 40, 40, 39, 40, 39, 39, 39, 39, 39, 40, 40, 40, 40, 39, 39, 40, 40, 39, 40, 40, 40, 40, 39, 40, 39, 39, 39, 40, 41, 41, 41, 40, 41, 41, 40, 40, 40, 41, 40, 41, 41, 40, 40, 41, 40, 41, 41, 41, 40, 40, 41, 41, 40, 40, 40, 40, 40, 40, 40, 40, 41, 40, 41, 40, 40, 40, 40, 40, 40, 41, 41, 40, 41, 40, 41, 40, 40, 40, 40, 40, 41, 41, 41, 40, 41, 41, 41, 40, 40, 41, 40, 40, 40, 40, 40, 40, 40, 41, 41, 41, 41, 41, 41, 41, 40, 40, 40, 41, 40, 40, 40, 40, 40, 40, 40, 41, 40, 40, 40, 40, 40, 40, 40, 40, 41, 40, 40, 40
```

Comment:
@pgmpablo157321 I am fine with this way of specifying the group sizes inside the mlperf.conf. I see this case it works because we use sample index from 0 to 6018 and the group sizes in this line is consistent with it.
What if we have different dataset that uses non-consecutive sample indices? Say first group is 0-40, second is 70-110, third is 120 to 159, etc? 
I don't think we need to touch the code up to handle this, but it's a good idea having this in mind for possible future issues.